python/ray/train/_internal/backend_executor.py

```diff
@@ -816,7 +816,7 @@ def __len__(self):
         raise InactiveWorkerGroupError()
 
 
-def _get_session(method_name: str):
+def _get_session_with_caller_info(method_name: str):
```

Comment:
What does `caller_info` refer to here?

Reply:
caller info is just the caller's `method_name`. I was doing a global find and replace to remove the `air._get_session` usage, and this one has the same name so I changed it. I can revert this change actually, since the global change is already done.